### PR TITLE
Set up mechanism to check dependencies' license compatibilities

### PR DESCRIPTION
**Pull request by kezhenxu94**
The compatible matrix needs further polishes, but his sets up the mechanism.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +137,33 @@ ERROR failed to identify the licenses of following packages:
 gopkg.in/yaml.v3
 ```
 
+#### Check Dependencies' licenses
```
Comment (wu-sheng) [inline README.md:140]: Should we provide PR comment after the check?

**Review by wu-sheng — APPROVED** [review]
Cool!

**Comment by wu-sheng** [comment]
Let's run for the source code itself?

**Comment by kezhenxu94** [comment]
> Let's run for the source code itself?

Some dependencies' licenses are still unrecognized, I need to polish and resolve them first.

**Pull request merged**